Dockerfile

```diff
@@ -20,7 +20,13 @@ RUN apt update \
     libpugixml-dev \
     libpcap0.8-dev \
     python \
+    apt-transport-https \
+    ca-certificates \
```

Comment:
Why is this needed?

Reply:
The `actions/chekcout@v2` needs those to checkout our repo. The `runs-on` images have it installed, so we need it inside our container as well for it to work. See this [failed build](https://github.com/cctechwiz-forks/netmeld/actions/runs/96588761) for an example of the error I was getting if you want more context.